src/network/networkd-ndisc.c

```diff
@@ -1176,29 +1176,37 @@ int config_parse_address_generation_type(
         if (r < 0)
                 return log_oom();
 
-        if ((p = startswith(rvalue, "static:")))
-                token->address_generation_type = IPV6_TOKEN_ADDRESS_GENERATION_STATIC;
-        else if ((p = startswith(rvalue, "prefixstable:")))
+        if ((p = startswith(rvalue, "prefixstable"))) {
                 token->address_generation_type = IPV6_TOKEN_ADDRESS_GENERATION_PREFIXSTABLE;
-        else {
+                if (*p == ':')
+                        p++;
+                else
+                        p = NULL;
+        } else {
                 token->address_generation_type = IPV6_TOKEN_ADDRESS_GENERATION_STATIC;
```

Comment:
Can you give a example of the format which is we trying to parse ?

Reply:
Examples:
```
IPv6Token=::1a:2b:3c:4d
IPv6Token=static:::1a:2b:3c:4d   <-- this is equivalent to the first one.
IPv6Token=prefixstable           <-- this is new.
IPv6Token=prefixstable:2002:da8:1::
```

Reply:
I'm specifically looking for `IPv6Token=prefixstable`, because I get my prefix from the router/ISP